test/WorkerChannelTests.ts

```diff
@@ -503,12 +570,7 @@ describe('WorkerChannel', () => {
         result:  {
           status: rpc.StatusResult.Status.Success
         },
-        outputData: [{
-          data: {
-            http: httpResponse
-          },
-          name: "res"
-        }],

```

Comment:
With this change, we will no longer see outputData being populated with "res", which is defined automatically, if there is a return value. this is ok because in cases where an http output is defined through return/context.done, it always takes priority